docs/blog/introduction-0.13.0/index.qmd

```diff
@@ -0,0 +1,270 @@
+    )
+)
+
+gt_tbl
+```
+
+Great! And a good strategy for future tables might be to style with combined location helpers first and then drill into the specific cells of those super locations with more fine-grained styles in a later `tab_style()` call.
+
+### Styling the title and the subtitle
+
+Although it really doesn't appear to have separate locations, the table header (produced by way of `tab_header()`) can have two of them: the title and the subtitle (the latter is optional). These can be targeted via `loc.title()` and `loc.subtitle()`. Let's focus in on the title location and add an assortment of styles to it.
+
+```{python}
+gt_tbl = (
+    gt_tbl
+    .tab_style(
+        style=[
+            style.text(size="24px"),
+            style.fill(color="aliceblue"),
+            style.borders(sides="bottom", color="#BFDFF6", weight="2px")
+        ],
+        locations=loc.title()
+    )
+)
+
+gt_tbl
+```
+
+Looks good. Notice that the title location is separate from the subtitle one, the background fill reveals the extent of its area.
+
+A subtitle is an optional part of the header. We do have one in our table example, so let's style that as well. The `style.css()` method will be used to give the subtitle text some additional top and bottom padding, and, we'll put in a fancy background involving a linear gradient.
+
+```{python}
+gt_tbl = (
+    gt_tbl
+    .tab_style(
+        style=style.css(rule="padding-top: 5px;"
+            "padding-bottom: 5px;"
+            "background-image: linear-gradient(120deg, #d4fc79 0%, #96f6a1 100%);"
+        ),
+        locations=loc.subtitle()
+    )
+)
+
+gt_tbl
+```
+
+None of what was done above could be done prior to `v0.13.0`. The `style.css()` method makes this all possible.
+
+The combined location helper for the title and the subtitle locations is `loc.header()`. As mentioned before, it can be used as a shorthand for `locations=[loc.title(), loc_subtitle()]` and it's useful here where we want to change the font for the title and subtitle text.
+
+```{python}
+gt_tbl = (
+    gt_tbl
+    .tab_style(
+        style=style.text(font=google_font("IBM Plex Serif")),
+        locations=loc.header()
+    )
+)
+
+gt_tbl
+```
+
+Though the order of things matters when setting styles via `tab_style()`, it's not a problem here to set a style for the combined 'header' location after doing so for the 'title' and 'subtitle' locations because the 'font' attribute *wasn't* set by `tab_style()` for those smaller locations.
+
+### How `tab_style()` fits in with `tab_options()`
+
+When it comes to styling, you can use `tab_options()` for some of the basics and use `tab_style()` for the more demanding styling tasks. And you could combine the usage of both in your table. The precedence (or priority) is given to `tab_style()`, regardless of the order of invocation. As an example of this, we can add a background color to all remaining locations with `tab_options(table_background_color=...)`:
```

Comment:
I would move this to after the example, with a short sentence explaining to people what is changing in the table (e.g. HoneyDew fill)

Reply:
This has been rewritten with the explanatory sentences after the example code.